certified-connectors/Documenatron Templating/readme.md

```diff
@@ -0,0 +1,28 @@
+## Documenatron Templating Connector
+Documenatron Templating is a fast, simple, and feature rich solution for automating the creation of office documents (docx, pptx and xlsx). It has connectors ready to go for Power Automate and Logic Apps, and can be integrated into any solution through an easy to use REST API.
+
+## Pre-requisites
+You will need the following to proceed:
+* A Microsoft Power Apps or Power Automate plan with custom connector feature
+* A valid [Documenatron Account](https://www.documenatron.com/signin) and API key.
+* The Power platform CLI tools
+
+### Deploying the sample
+Run the following commands and follow the prompts:
+
+```paconn
+paconn create --api-def apiDefinition.swagger.json --api-prop apiProperties.json
+```
+#### Manual upload
+Use the [Power Automate Portal](https://flow.microsoft.com) to create a new custom connector. Use **Import an OpenAPI file** and select the apiDefinition.swagger.json file.
+
+### Connector Security setup
+
+Use the following values for the the Oath 2.0 security page:
+
+* `Authentication type` : API Key
+* `API Key` provided through your Documenatron account.
+
```

Comment:
Known issue and Deployment Instructions section are missing in the readme.md file.  Please refer this link and update duly. https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md